package_test.go

```diff
@@ -1383,6 +1531,7 @@ func (s *PackageSuite) TestRaceConditionFinalizer(c *C) {
 	// Assert that sql.Stmt was not closed early.
 	c.Assert(q.Run(), IsNil)
 }
+
```

Comment:
The missing newline here was intentional :) so that the comment above applied to the two tests.

Reply:
Added back in.